### PR TITLE
added numeric IDs to all words in dictionary / changed filter functio…

### PR DESCRIPTION
**Pull request by LipinVV**
…n / chained new dictionary with practice block

### CODE REVIEW

**Comment by LipinVV** [comment]
1) When a dictionary is loaded - words are not rendered. Need to understand how to implement filter function in a proper way;
2) When pressing on navigation buttons during matching contest - options of contest re-renders in a loop of 4 of them;
3) Need to understand how to add audio files to the dictionary and other contests. Add a button to prevent words auto-playing;
4) Implement a feature when you've got an option to train a specific number of saved words collection uploaded from dictionary;
5) Implement a route between practice section and sub-sections: type-contest/match-contest/other new.

File: .eslintcache [hunk 1]
```diff
@@ -1 +1 @@
...
...,"endLine":23,"endColumn":31},{"ruleId":"117","severity":1,"message":"122","line":3,"column":8,"nodeType":"119","messageId":"120","endLine":3,"endColumn":16},{"ruleId":"117","severity":1,"message":"123","line":19,"column":15,"nodeType":"119","messageId":"120","endLine":19,"endColumn":21},"no-native-reassign",["124"],"no-negated-in-lhs",["125"],"array-callback-return","Array.prototype.filter() expects a value to be returned at the end of arrow function.","ArrowFunctionExpression","expectedAtEnd",["124"],["125"],"react-hooks/exhaustive-deps","React Hook useEffect has a missing dependency: 'words'. Either include it or remove the dependency array. You can also replace multiple useState variables with useReducer if 'setFiltered' needs the current value of 'words'.","ArrayExpression",["126"],"no-unused-vars","'ReactPlayer' is defined but never used.","Identifier","unusedVar","'setRunning' is assigned a value but never used.","'firebase' is defined but never used.","'newDoc' is assigned a value but never used.","no-global-assign","no-unsafe-negation",{"desc":"127","fix":"128"},"Update the dependencies array to be: [search, words]",{"range":"129","text":"130"},[945,953],"[search, words]"]
```
Comment (ilyapasyuk) [inline .eslintcache:1]: Этот файл надо бы добавить в .gitignore

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
-import React from 'react'
+import { React } from 'react'
```
Comment (ilyapasyuk) [inline src/App.js:1]: Мне кажется, тут с импортом ошибка 🧐

File: src/components/CreateWord/createword.js [hunk 3]
```diff
@@ -5,99 +5,45 @@ import firebase, { db } from '../../services/firebase'
...
     return (
         <div className=''>
-            <button type='button' onClick={getDictionary}>GET</button>
             <form className='word__creator'>
                 <label className='word__label'>Слово
                     <input
                         type='text'
                         value={word}
-                        onChange={typeWord}
+                        onChange={() => setWord()}
```
Comment (ilyapasyuk) [inline src/components/CreateWord/createword.js:34]: Для справки - можно ещё так написать onClick={setWord}

File: src/components/practice/practice.js [hunk 4]
```diff
@@ -1,17 +1,31 @@
 import React, { useEffect, useState } from 'react'
-import { dictionary } from '../dictionary/data'
-
+import { GetDictionary } from "../../services/dictionary";
 
 export const Practice = () => {
+    // можно ли прокинуть из одного файла во все?
```
Comment (ilyapasyuk) [inline src/components/practice/practice.js:5]: Да, с помощью контекста

**Pull request merged**